doc/source/dev/index.rst

```diff
@@ -17,6 +17,7 @@ the organization section.
    dev_quickstart
    contributor/development_workflow
    contributor/contributor_toc
+   contributor/triage
```

Comment:
I am not sure about having this as a top level. Issue triaging is to me as important as PR review.

Reply:
Ok - moved it into the contributor toc, let me know if this is better.